disperser/dataapi/metrics_handlers.go

```diff
@@ -142,3 +143,62 @@ func (s *server) getNonSigners(ctx context.Context, intervalSeconds int64) (*[]N
 
 	return &nonSignersObj, nil
 }
+
+func (s *server) getUnsignedBatches(ctx context.Context, intervalSeconds int64) (*OperatorNonsigningPercentage, error) {
+	nonSigners, err := s.subgraphClient.QueryBatchNonSigningOperatorIdsInInterval(ctx, intervalSeconds)
+	if err != nil {
+		return nil, err
+	}
+
+	var (
+		limit        = 1000
+		skip         = 0
+		totalBatches = 0
+	)
+
+loop:
+	for {
+		select {
+		case <-ctx.Done():
+			break loop
+		default:
+			batches, err := s.subgraphClient.QueryBatchesWithLimit(ctx, limit, skip)
+			if err != nil {
+				s.logger.Error("Failed to query batches", "error", err)
+				return nil, err
+			}
+			totalBatches += len(batches)
+			if len(batches) == 0 {
+				break
+			}
+			skip += limit
+		}
+	}
+
+	percentage := make(map[string]float64, 0)
+
+	if totalBatches < 1 {
+		return &OperatorNonsigningPercentage{
+			TotalNonSigners:       len(nonSigners),
+			TotalBatches:          totalBatches,
+			PercentagePerOperator: percentage,
+		}, nil
+	}
+
+	for operatorId, totalUnsignedBatches := range nonSigners {
+		ps := fmt.Sprintf("%.2f", float64(totalUnsignedBatches)/float64(totalBatches))
+		pf, err := strconv.ParseFloat(ps, 64)
+		if err != nil {
+			return nil, err
+		}
+		if pf > 0 {
```

Comment:
Can we just use `totalUnsignedBatches > 0`? It's integer, so play better with comparison.